bin/process.sh

```diff
@@ -6,10 +6,15 @@ shift $[$#-2]
 
 input=$1
 output=$2
-# target size in KB
-THUMBNAIL_SIZE=${THUMBNAIL_SIZE:-300}
+
+THUMBNAIL_SIZE=${THUMBNAIL_SIZE:-300} # target size in KB
 TILER_BASE_URL=${TILER_BASE_URL:-http://tiles.openaerialmap.org}
 
+# Although TMPDIR is usually set by the OS, it doesn't seem to be getting passed through
+# Docker and Node's `spawn()`, so no harm done in just defaulting to `/tmp` in such cases.
+# There may be clues here as the underlying issue: https://github.com/npm/npm/issues/4531
+TMPDIR={TMPDIR:-/tmp}
```

Comment:
We were seeing files written and then disappearing before being read. It was particularly noticeable on large EC2 instances being used for multiple batch jobs.

Agreed on doc'ing the space requirement. This _should_ clean up after itself properly, so a normal directory could work (and if it doesn't, I'd consider that a bug).